docs/source/reference/technical-overview.md

```diff
@@ -72,20 +71,20 @@ When a user accesses JupyterHub, the following events take place:
 - A single-user notebook server instance is [spawned](./spawners.md) for the
   logged-in user
 - When the single-user notebook server starts, the proxy is notified to forward
-  requests to `/user/[username]/*` to the single-user notebook server.
-- A cookie is set on `/hub/`, containing an encrypted token. (Prior to version
+  requests made to `/user/[username]/*`, to the single-user notebook server.
+- A [cookie](https://en.wikipedia.org/wiki/HTTP_cookie) is set on `/hub/`, containing an encrypted token. (Prior to version
   0.8, a cookie for `/user/[username]` was used too.)
 - The browser is redirected to `/user/[username]`, and the request is handled by
   the single-user notebook server.
 
-The single-user server identifies the user with the Hub via OAuth:
+How does the single-user server identify the user with the Hub via OAuth?
 
-- on request, the single-user server checks a cookie
-- if no cookie is set, redirect to the Hub for verification via OAuth
-- after verification at the Hub, the browser is redirected back to the
+- On request, the single-user server checks a cookie
+- If no cookie is set, it redirects to the Hub for verification via OAuth
```

Comment:
Maybe a good idea to minimize pronouns when there are so many things being talked about: 

```suggestion
- If no cookie is set, the single-user server redirects to the Hub for verification via OAuth
```

Even though it's redundant with the previous bullet, in this particular doc I think it's good to minimize possible sources of "Wait, what is 'it' here?"